bindings/python/tests/test_offline.py

```diff
@@ -14,6 +14,7 @@
 client = Client()
 
 
+@pytest.mark.skip(reason="temporarily skipped until 2.0 blocks in use")
```

Comment:
?

Reply:
was overly specific with the reason and c&p'ed